pkg/tracing/logger_test.go

```diff
@@ -0,0 +1,21 @@
+package tracing
+
+import "testing"
+
+func TestExtractFuncName(t *testing.T) {
+	for _, test := range []struct {
+		input    string
+		expected string
+		ok       bool
+	}{
+		{"", "", false},
+		{"porter/", "", false},
+		{"porter/v.", "", false},
+		{"github.com/getporter/porter/tracing.StartSpan", "StartSpan", true},
+	} {
+		fn, ok := extractFuncName(test.input)
+		if fn != test.expected || ok != test.ok {
```

Comment:
You don't need to change this test, it's fine. But I did want to make you aware that we use github.com/stretchr/testify a lot to help with our test asserts. It helps craft the error message of a failed test so that you don't have to, and has a lot of useful built in assertions.